ui/src/components/role-policy/AddRuleFormForRole.js

```diff
@@ -167,7 +175,18 @@ export default class AddRuleFormForRole extends React.Component {
                             name={'rule-resource' + id}
                             value={this.state.resource}
                             onChange={resourceChanged}
-                            placeholder={'Rule Resource (Case Sensitive)'}
+                            placeholder={'Rule Resource'}
+                        />
+                    </ContentDiv>
+                </SectionDiv>
+                <SectionDiv>
+                    <ContentDiv>
+                        <CheckBox
```

Comment:
have to give a unique key to the checkbox component 
https://reactjs.org/docs/lists-and-keys.html#keys
This is one option to generate a unique id : https://www.geeksforgeeks.org/lodash-_-uniqueid-method/

Reply:
also to check the effect of it, in the policy page open add rule forms of two different policies and try clicking on the checkbox.

Reply:
Fixed